src/Razor/src/Microsoft.VisualStudio.RazorExtension.Dependencies/Microsoft.VisualStudio.RazorExtension.Dependencies.csproj

```diff
@@ -48,9 +48,12 @@
     <PackageReference Include="Microsoft.VisualStudio.LanguageServer.Protocol" Version="$(MicrosoftVisualStudioLanguageServerProtocolPackageVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.LanguageServer.Protocol.Extensions" Version="$(MicrosoftVisualStudioLanguageServerProtocolExtensionsPackageVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.LanguageServer.Protocol.Internal" Version="$(MicrosoftVisualStudioLanguageServerProtocolInternalPackageVersion)" />
+    <PackageReference Include="Microsoft.VisualStudio.LanguageServer.Client.Implementation" Version="$(MicrosoftVisualStudioLanguageServerClientImplementationPackageVersion)" />
   </ItemGroup>
 
   <ItemGroup>
+    <VSIXSourceItem Include="$(OutputPath)Microsoft.VisualStudio.LanguageServer.Client.dll" />
+    <VSIXSourceItem Include="$(OutputPath)Microsoft.VisualStudio.LanguageServer.Client.Implementation.dll" />
```

Comment:
Why are these needed?

Reply:
In the end, this probably isn't. I was chasing down dependencies that we had whose version was newer than what is in 17.7 P1, and these two DLLs were present as load failures in the ActivityLog. Sadly adding these didn't resolve the issues fully, so I ended up turning off `IncludeRoslynDeps` and hence these are strictly unnecessary for this PR. This is about where I stopped though, because clearly we need to discuss this PR further. I see you've started that on Teams though :)